test/ui_test_suite.rb

```diff
@@ -567,7 +580,7 @@ def open_new_page(url)
 	end
 
 	#create a 2d scatter study for use in user annotation testing
-	test 'admin: create a 2d scatter study' do
```

Comment:
I like what you did here with the namespacing, but they should still have front-end: as part of the name so that we can run all front end tests as needed.  We can have as many namespaces as we want, and we can nest them.